media-api/app/controllers/AggregationController.scala

```diff
@@ -11,6 +11,8 @@ class AggregationController(auth: Authentication, elasticSearch: ElasticSearchVe
   extends BaseController with AggregateResponses {
 
   def dateHistogram(field: String, q: Option[String]) = auth.async { request =>
+    implicit val r = request
```

Comment:
I think you can inline this above:

```scala
  def dateHistogram(field: String, q: Option[String]) = auth.async { implicit request =>
```

Reply:
Probably; if none of the downstream code already references the explicit request; hence I'm providing both.